src/tests/Login/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render } from '@testing-library/react';
+import LoginScreen from '../../screens/Login';
+
+describe('<LoginScreen />', () => {
```

Comment:
How about just `LoginScreen`?
```suggestion
describe('LoginScreen', () => {
```

Reply:
✅ Done in 39a594b :)